new/LIT6215MGChurchProp.xml

```diff
@@ -0,0 +1,103 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6215MGChurchProp" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: St George punishes a man who
+                    stole church’s property</title>
+                <title xml:id="t2" xml:lang="en">How many people were cured in the church of St
+                    George</title>
```

Comment:
numerous rather than many

Reply:
Same as above: title as given by Strelcyn

Reply:
"How many" sounds like the beginning of a question, but again, this is not life or death :)

Reply:
Titel 2 und title 3 are takem from other catalogues. I copied the title as written in the catalogue. I would propose to leave it as such, even though it sounds strange.

Reply:
Perfect, we can always pick this up again at the end in a broader way if we feel it's necessary.